nipype/interfaces/fsl/utils.py

```diff
@@ -1702,74 +1730,75 @@ class ConvertWarpInputSpec(FSLCommandInputSpec):
                   desc='filename for pre-transform (affine matrix)')
 
     warp1 = File(exists=True, argstr='--warp1=%s',
-                 desc=('Name of file containing initial warp-fields/coefficients (follows premat). This could e.g. be a '
-                       'fnirt-transform from a subjects structural scan to an average of a group '
-                       'of subjects.'))
+                 desc='Name of file containing initial warp-fields/coefficients (follows premat). This could e.g. be a '
```

Comment:
are you sure, this works? without the parenthesis? can you check the doc rendering as well?